rust/android/Dockerfile

```diff
@@ -84,9 +85,11 @@ RUN cd /opt/openssl-3.0.1 && \
     make -j$(nproc) && make -j$(nproc) install && \
     make clean && make distclean
 
-RUN rustup component add rustfmt
-
-RUN cd /opt && git clone --depth=1 https://github.com/zingolabs/zecwallet-mobile zecwalletmobile && cd zecwalletmobile/rust/android && cargo fetch && cargo build --release && rm -rf /opt/zecwalletmobile
+RUN cd /opt && git clone --depth=1 --branch="dev_working" https://github.com/dannasessha/zecwallet-mobile zecwalletmobile
```

Comment:
This line still points to my own repo, which I needed to do to test changes without polluting repos here on `zingolabs`. When this merges in, we should do a follow-up PR to move everything to `zingolabs`.

Reply:
Fine.